config/crd/bases/postgres-operator.crunchydata.com_pgadmins.yaml

```diff
@@ -1949,6 +1949,18 @@ spec:
                         items:
                           type: string
                         type: array
+                      retentionPeriod:
+                        description: |-
+                          How long to retain log files locally. An RFC 3339 duration or a number
+                          and unit: `3d`, `4 weeks`, `12 hr`, etc.
+                        format: duration
+                        minLength: 1
+                        pattern: ^(PT)?(0|[0-9]+ *(?i:(h|hr|d|w|wk)|(hour|day|week)s?))+$
+                        type: string
+                        x-kubernetes-validations:
+                        - message: must be greater than one hour
+                          rule: self == duration("0") || (self >= duration("1h") &&
+                            self <= duration("8760h"))
```

Comment:
🤔 More like "must be at least one hour," eh?

Reply:
I kinda feel like if we let someone out 0, then we should respect it.

Or maybe we can warn?

Reply:
Or just not allow it?

Reply:
"smallest duration increment is one hour" :thinking:

Reply:
yeah, okay. i'm tired of zero now. taking it out.